### ISSUE TITLE
Defaults of naiveRCI

### ISSUE THREAD

**Issue by p-smirnov** [bhklab/fastCI#5]
While I understand the idea of having 0.2 as the default of naiveRCI (since its rCI), should we actually have 2 different functions for CI and rCI? 

Also, 0.2 default clashes with other implementations in lab, which default to CI. 
@iamsinht thoughts?

**Comment by iamsinht** [comment]
Yeah, that's a good point; I will deprecate naiveCI and rename naiveRCI with a default to 0.  Eventually, I will add (naive) kernel functionality.
